### DISCUSSION TITLE
Hints with letters

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#10101] [Q&A]
May I ask what these letters in the bottom left corner indicate and how can someone activate them?
Thanks!

![εικόνα](https://user-images.githubusercontent.com/76203123/217781082-c690cac5-a0cd-4b2f-a969-fea769bf0818.png)

**Comment by Frenzie** [comment]
Normally those should only show on a system with a keyboard, in which case "R" and "T" are the buttons to press on said keyboard to quickly activate the entry in question.

**Reply by ichnilatis-gr** [reply]
Oh, I see! Okay then... :-)

**Reply by Frenzie** [reply]
But where are you seeing them? Somewhere you shouldn't?

**Reply by ichnilatis-gr** [reply]
In various screenshots in the forum.
